index.html

```diff
@@ -0,0 +1,352 @@
+                         <div class="card-footer">
+                            <a href="" class="text-btn">Checkout</a>
+                            <a href="" class="text-btn">Buy</a>
+                            <a href=""><i class="fa fa-heart card-icon"></i></a>
+                            <a href=""><i class="fa fa-share-alt card-icon"></i></a>
+                            <a href=""><i class="fa fa-ellipsis-v card-icon"></i></a>           
+                            </div>
+                    
+                    </div>
+                
+                </div>
+    
+                <div class="col-sm-3">
+    
+                    <div class="card card-shadow ">
+                        <div class="card-header ">
+                            <div class="card-title text-center">Glory Phantom</div>
+                           
+                        </div>
+                        <div height="400">
+                        <img src="assets/img/list_img/6.jpg"  class="card-image img-c" alt="Card-Image">
+                         </div>
+                         <div class="card-footer">
+                            <a href="" class="text-btn">Checkout</a>
+                            <a href="" class="text-btn">Buy</a>
+                            <a href=""><i class="fa fa-heart card-icon"></i></a>
+                            <a href=""><i class="fa fa-share-alt card-icon"></i></a>
+                            <a href=""><i class="fa fa-ellipsis-v card-icon"></i></a>           
+                            </div>
+                       
+                    </div>
+    
+                </div>
+    
+                <div class="col-sm-3">
+    
+                    <div class="card card-shadow ">
+                        <div class="card-header">
+                            <div class="card-title text-center">Glory Black Stone</div>
+                        </div>
+    
+                        <div height="400">
+                        <img src="assets/img/list_img/7.jpg"  class="card-image img-c" alt="Card-Image">
+                        </div>
+                        <div class="card-footer">
+                            <a href="" class="text-btn">Checkout</a>
+                            <a href="" class="text-btn">Buy</a>
+                            <a href=""><i class="fa fa-heart card-icon"></i></a>
+                            <a href=""><i class="fa fa-share-alt card-icon"></i></a>
+                            <a href=""><i class="fa fa-ellipsis-v card-icon"></i></a>           
+                            </div>
+                    </div>
+    
+                </div>
+    
+                <div class="col-sm-3">
+                    <div class="card card-shadow ">
+                        <div class="card-header">
+                            <div class="card-title text-center">Ray Phantom</div>
+                        </div>
+                        
+                        <div height="400">
```

Comment:
what is height here? why is the value of 400 given for this

Reply:
done what you told me .